fmriprep/interfaces/reports.py

```diff
@@ -55,26 +68,30 @@ def _run_interface(self, runtime):
         return runtime
 
 
-class AnatomicalSummaryInputSpec(BaseInterfaceInputSpec):
+class SubjectSummaryInputSpec(BaseInterfaceInputSpec):
     t1w = InputMultiPath(File(exists=True), desc='T1w structural images')
+    t2w = InputMultiPath(File(exists=True), desc='T2w structural images')
     subjects_dir = Directory(desc='FreeSurfer subjects directory')
-    subject_id = traits.Str(desc='FreeSurfer subject ID')
+    subject_id = traits.Str(desc='Subject ID')
```

Comment:
Use `Str` from `nipype.interfaces.base` better than `traits.Str`? I think we try to shadow `traits.Str` but it is safer to use the earlier (and it is already imported in this file...)